components/permissions/permission_widevine_utils.cc

```diff
@@ -0,0 +1,31 @@
+/* Copyright (c) 2023 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/permissions/permission_widevine_utils.h"
+
+#include "brave/components/constants/pref_names.h"
+#include "components/permissions/permission_request.h"
+#include "components/permissions/request_type.h"
+#include "components/prefs/pref_service.h"
+
+namespace permissions {
+
+void DontAskWidevineInstall(PrefService* prefs, bool dont_ask) {
```

Comment:
this is extremely confusing imo to invert the pref from "ask" to "don't ask" and the helper seems unnecessary

Reply:
The function used on desktop is called `DontAskWidevineInstall` so I just followed it. I can make it the other way though.

Reply:
yes, I did notice later that other code does that as well, I think we should be consistent with `Ask` instead of `DontAsk` to match the pref

Reply:
I changed `DontAskWidevineInstall` to `AskWidevineInstall` and `dont_ask` to `ask`. All the calls are reverted.